src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -287,7 +287,7 @@ public void UpdateValue(string name, object value, Type type)
         {
             Debug.Assert(null != name, "[SerializationInfo.UpdateValue]name!=null");
             Debug.Assert(null != value, "[SerializationInfo.UpdateValue]value!=null");
```

Comment:
I get that we're only replacing Object.ReferenceEquals, but to my eyes, it becomes less obvious why the lines above are still `!=` instead of `is not`

Reply:
> less obvious why

Because fixing up all of those is a much larger change (a different rule).  Baby steps.